shared/style/edit_mode.css

```diff
@@ -15,27 +15,20 @@
 }
 
 /* Header */
-[role="dialog"][data-type="edit"] [role="region"] > header:first-child,
 [role="dialog"][data-type="edit"] [role="region"] > gaia-header {
   background-color: #f4f4f4;
   transition: transform 0.4s ease;
   pointer-events: auto;
 }
 
-[role="dialog"][data-type="edit"] [role="region"] > header:first-child h1,
 [role="dialog"][data-type="edit"] [role="region"] > gaia-header h1 {
   color: #868692;
 }
 
-[role="dialog"][data-type="edit"] [role="region"] > header:first-child menu[type="toolbar"] button,
 [role="dialog"][data-type="edit"] [role="region"] > gaia-header button {
   color: #00aac5;
 }
 
-[role="dialog"][data-type="edit"] [role="region"] > header:first-child .icon.icon-close {
```

Comment:
Looks like all these removed selectors are required only in "example" file [1] and don't have any value for real usages. Probably we'll need to confirm with Pavel if it's ok that [1] is affected :)

[1]  ```/shared/style/edit_mode/index.html```